packages/driver-adapter-utils/src/types.ts

```diff
@@ -115,7 +154,8 @@ export type ConnectionInfo = {
   maxBindValues?: number
 }
 
-export type Flavour = 'mysql' | 'postgres' | 'sqlite'
+// TODO: I don't really like that we call this "flavour" in some places and "provider" in others. :/
+export type Flavour = 'mysql' | 'postgres' | 'sqlite' | 'mongodb'
```

Comment:
I feel you: https://github.com/prisma/prisma/issues/21865

Reply:
So our goal state is to ideally call it "provider" everywhere?